@jpdavis-prof - I'm attempting to test a node-images change and when building CSM with image 7.2.2, I'm getting an RPM signing key error with nvidia-external
```
20:19:19  2025-10-08 01:19:19,447	INFO	Downloading /data/nvidia-external/mellanox/15-SP7/x86_64/kernel-mft-mlnx-kmp-default-4.33.0_k6.4.0_150700.51-1.sles15sp7.x86_64.rpm
20:19:19  2025-10-08 01:19:19,471	INFO	Verifying signature for kernel-mft-mlnx-kmp-default-4.33.0_k6.4.0_150700.51-1.sles15sp7.x86_64.rpm against provided GPG keys
20:19:19  2025-10-08 01:19:19,483	ERROR	
20:19:19  2025-10-08 01:19:19,483	ERROR	Signature verification failed for /data/nvidia-external/mellanox/15-SP7/x86_64/kernel-mft-mlnx-kmp-default-4.33.0_k6.4.0_150700.51-1.sles15sp7.x86_64.rpm
20:19:19  2025-10-08 01:19:19,483	ERROR	
20:19:19  2025-10-08 01:19:19,483	ERROR	Stdout:
20:19:19  2025-10-08 01:19:19,483	ERROR	/data/nvidia-external/mellanox/15-SP7/x86_64/kernel-mft-mlnx-kmp-default-4.33.0_k6.4.0_150700.51-1.sles15sp7.x86_64.rpm:
20:19:19  2025-10-08 01:19:19,483	ERROR	    Header V4 RSA/SHA256 Signature, key ID e6d6a281: NOKEY
20:19:19  2025-10-08 01:19:19,483	ERROR	    Header SHA256 digest: OK
20:19:19  2025-10-08 01:19:19,483	ERROR	    Header SHA1 digest: OK
20:19:19  2025-10-08 01:19:19,483	ERROR	    Payload SHA256 digest: OK
20:19:19  2025-10-08 01:19:19,483	ERROR	    V4 RSA/SHA256 Signature, key ID e6d6a281: NOKEY
20:19:19  2025-10-08 01:19:19,483	ERROR	    MD5 digest: OK
20:19:19  2025-10-08 01:19:19,483	ERROR	
20:19:19  2025-10-08 01:19:19,483	ERROR	
20:19:19  2025-10-08 01:19:19,483	ERROR	Stderr:
20:19:19  2025-10-08 01:19:19,483	ERROR	
20:19:19  2025-10-08 01:19:19,483	ERROR	Error downloading from ['https://artifactory.algol60.net/artifactory/nvidia-external/mellanox/15-SP7/x86_64/kernel-mft-mlnx-kmp-default-4.33.0_k6.4.0_150700.51-1.sles15sp7.x86_64.rpm']
20:19:19  Traceback (most recent call last):
20:19:19    File "/usr/local/bin/rpm-sync", line 273, in download_asset
20:19:19      unsigned_rpm = do_verify_rpm_signature(path, keys)
20:19:19                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
20:19:19    File "/usr/local/bin/rpm-sync", line 229, in do_verify_rpm_signature
20:19:19      sys.exit(1)
20:19:19  SystemExit: 1
20:19:19  2025-10-08 01:19:19,493	ERROR	Task exception was never retrieved
20:19:19  future: <Task finished name='Task-144' coro=<download_asset() done, defined at /usr/local/bin/rpm-sync:244> exception=SystemExit(1)>
20:19:19  Traceback (most recent call last):
20:19:19    File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
20:19:19      return runner.run(main)
20:19:19             ^^^^^^^^^^^^^^^^
20:19:19    File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
20:19:19      return self._loop.run_until_complete(task)
20:19:19             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
20:19:19    File "/usr/local/lib/python3.11/asyncio/base_events.py", line 640, in run_until_complete
20:19:19      self.run_forever()
20:19:19    File "/usr/local/lib/python3.11/asyncio/base_events.py", line 607, in run_forever
20:19:19      self._run_once()
20:19:19    File "/usr/local/lib/python3.11/asyncio/base_events.py", line 1922, in _run_once
20:19:19      handle._run()
20:19:19    File "/usr/local/lib/python3.11/asyncio/events.py", line 80, in _run
20:19:19      self._context.run(self._callback, *self._args)
20:19:19    File "/usr/local/bin/rpm-sync", line 273, in download_asset
20:19:19      unsigned_rpm = do_verify_rpm_signature(path, keys)
20:19:19                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
20:19:19    File "/usr/local/bin/rpm-sync", line 229, in do_verify_rpm_signature
20:19:19      sys.exit(1)
20:19:19  SystemExit: 1
```
I think a gpg key needs to be added to Artifactory in https://artifactory.algol60.net/artifactory/gpg-keys and then that key added to the `HPE_RPM_SIGNING_KEYS` list in csm `assets.sh`.

--- a/assets.sh
+++ b/assets.sh
@@ -55,6 +55,7 @@ COMPUTE_IMAGE_ID=7.2.2
 # suse_ptf_key.asc - for SUSE PTF kernel packages, see https://www.suse.com/support/kb/doc/?id=000018545
 # opensuse-tumbleweed.asc - k8s packages taken from https://download.opensuse.org/tumbleweed/repo/oss/repodata/
 # isv_kubernetes_key.gpg - k8s packages taken from https://build.opensuse.org/project/subprojects/isv:kubernetes (expires 2026-12-29)
+# nvidia-mellanox-1.gpg - for Nvidia Mellanox OFED packages in nvidia-external Artifactory repo
 HPE_RPM_SIGNING_KEYS=(
     https://artifactory.algol60.net/artifactory/gpg-keys/hpe-signing-key.asc
     https://artifactory.algol60.net/artifactory/gpg-keys/hpe-signing-key-fips.asc
@@ -67,4 +68,5 @@ HPE_RPM_SIGNING_KEYS=(
     https://artifactory.algol60.net/artifactory/gpg-keys/suse_ptf_key.asc
     https://artifactory.algol60.net/artifactory/gpg-keys/opensuse-tumbleweed.asc
     https://artifactory.algol60.net/artifactory/gpg-keys/isv_kubernetes_key.gpg
+    https://artifactory.algol60.net/artifactory/gpg-keys/nvidia-mellanox-1.gpg
 )

